.github/workflows/ci-master.yml

```diff
@@ -91,7 +91,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v1
         with:
-          python-version: 3.8
+          python-version: 3.7
```

Comment:
Since this has been changed to 3.7, you will also need to update the below values to 3.7. That should fix the CI issues on windows. 
e.g.
`tox -e py38 --notest`
to 
`tox -e py37 --notest`

Reply:
thanks, protobuf binary weren't being published for python 3.8